### DISCUSSION TITLE
Welcome to RanseiLink Discussions!

### DISCUSSION THREAD

**Discussion by Deijin27** [Deijin27/RanseiLink#27] [Announcements]
## 👋 Welcome!

Feel free to ask questions or share ideas (Click the green button that says "New Discussion")

Also check out the [Ransei Region Discord Server](https://discord.gg/NSCJ4yuRfa)

**Comment by WhatAUsernameIHave** [comment]
Suggestions for the Max Link Table

1. Add another module that has the inverse and shows every Warrior for a Pokémon.
2. Have an option that allows you to copy changes to other Pokemon in the evolution line. 
3. Add a filter option that allows you to show/hide different levels of links. For example, you click hide then input 70% into a text box and it’ll hide any Warrior/Pokémon that is 69% or lower.

**Reply by Deijin27** [reply]
Absolutely. The first two options sound good.

The last, perhaps could be better done with sorting methods. I've already thought that its a bit hard to know where in the order a pokemon is based on their ids, maybe it could have multiple sort options like name, value. I've also been going all out with including pokemon and warrior icons everywhere in the scenario-warrior-workspace. I will definitely add them here too.
